### PR TITLE
fixes #2623 - trends link was broken

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/views/trends/index.html.erb [hunk 1]
```diff
@@ -40,8 +40,8 @@
   <% end %>
 </table>
 
-<%= page_entries_info @trend, :model => "trends" %>
-<%= will_paginate @trend %>
+<%= page_entries_info @trends, :model => "trends" %>
```
Comment (abenari) [inline app/views/trends/index.html.erb]: while at it can you please also remove the :model => "trends" which is not needed any more.

**Comment by isratrade** [comment]
@abenari, any other changes need to be here?

**Comment by isratrade** [comment]
@abenari - re-commited

**Comment by abenari** [comment]
merged thanks

**Pull request closed without merging**